pkg/utils/log_utils.go

```diff
@@ -35,8 +35,13 @@ func PrintErrorInColor(message string) {
 	_, _ = messageColor.Fprint(os.Stderr, message)
 }
 
+// PrintfMessageToTUI prints the message to the stderr.
+func PrintfMessageToTUI(message string, args ...any) {
```

Comment:
This function doesn't belong in `log_utils`, which should be about logging. 

Actually, none of these belong here, because they are related to TUI.

- PrintMessage(message string)
- PrintMessageInColor(message string, messageColor *color.Color)
- PrintErrorInColor(message string)
- PrintfMessageToTUI(message string, args ...any)

Reply:
Maybe we can open a new PR that uses the new, non-deprecated functions. Then move `PrintfMessageToTUI` to a TUI package, so we can call `tui.Printf`.

Reply:
Yep that makes sense to me but yaa let us create a new pr for this one. I am creating a ticket to do thsi and linking it here.